package.json

```diff
@@ -53,49 +53,50 @@
   "devDependencies": {
     "@commitlint/cli": "^8.2.0",
     "@commitlint/config-angular": "^8.2.0",
-    "@testing-library/react": "^9.3.0",
-    "@types/jest": "^23.3.2",
-    "@types/node": "^10.11.0",
-    "@types/react": "^16.9.19",
+    "@testing-library/react": "^9.4.1",
+    "@types/jest": "^25.1.3",
+    "@types/node": "^13.7.7",
+    "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
-    "colors": "^1.3.2",
-    "coveralls": "^3.0.2",
-    "cross-env": "^5.2.0",
+    "colors": "^1.4.0",
+    "coveralls": "^3.0.9",
+    "cross-env": "^7.0.0",
     "husky": "^3.0.9",
-    "jest": "^23.6.0",
-    "jest-config": "^23.6.0",
+    "jest": "^25.1.0",
+    "jest-config": "^25.1.0",
     "jest-dom": "^4.0.0",
     "lodash.camelcase": "^4.3.0",
-    "prettier": "^1.14.3",
+    "prettier": "^1.19.1",
     "pretty-quick": "^2.0.0",
     "prompt": "^1.0.0",
-    "rimraf": "^2.6.2",
-    "rollup": "^0.67.0",
-    "rollup-plugin-commonjs": "^9.1.8",
-    "rollup-plugin-json": "^3.1.0",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "rollup-plugin-sourcemaps": "^0.4.2",
-    "rollup-plugin-typescript2": "^0.18.0",
-    "semantic-release": "^15.9.16",
-    "ts-jest": "^23.10.2",
-    "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
-    "tslint-config-prettier": "^1.15.0",
-    "tslint-config-standard": "^8.0.1",
-    "typedoc": "^0.16.9",
-    "typescript": "^3.7.5",
-    "uuid": "^3.3.3"
+    "rimraf": "^3.0.2",
+    "rollup": "^1.32.0",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-json": "^4.0.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-sourcemaps": "^0.5.0",
+    "rollup-plugin-typescript2": "^0.26.0",
+    "semantic-release": "^17.0.2",
+    "ts-jest": "^25.2.1",
+    "ts-node": "^8.6.2",
+    "tslint": "^6.0.0",
+    "tslint-config-prettier": "^1.18.0",
+    "tslint-config-standard": "^9.0.0",
+    "typedoc": "^0.16.11",
+    "typescript": "^3.8.3",
+    "uuid": "^7.0.1"
   },
   "peerDependencies": {
-    "azure-maps-control": "2.0.21",
+    "azure-maps-control": "2.0.23",
```

Comment:
we are not ready for this version

Reply:
As I test in our playground, everything works fine with this version, let's call it as a proposal and check if everything works as expected